skymp5-client/src/front/browser.ts

```diff
@@ -7,76 +7,85 @@ import {
   settings,
```

Comment:
Code became less readable. Please find a way to configure bindings as you want without bloating up the code.